.env.example

```diff
@@ -6,7 +6,7 @@ TF_VAR_DISK_SIZE_GB=100# size of the disk in GB (default is 100GB)
 TF_VAR_ARCHIVE_DISK_SIZE_GB=100 # size of the disk in GB in archive node (default is 100GB)
 TF_VAR_IOPS=3000# Amount of provisioned IOPS
 TF_VAR_ARCHIVE_IOPS=15000 # Amount of provisioned IOPS in archive node
-TF_VAR_VALIDATOR_COUNT=2# number of validator nodes (default is 2)
+TF_VAR_VALIDATOR_COUNT=2# number of validator nodes (default is 2). Note that while spinning up a public network (mainnet/mumbai) node, this will serve as a non-validator
```

Comment:
why don't we use `TF_VAR_SENTRY_COUNT ` then?
This is confusing and errors prone

Reply:
Well , that's why I added the comment for clarification, thinking of it as a "special case". For mumbai/mainnet we'll anyway end up spinning a non-validator node

Reply:
Then let's use the sentry, no? 
Otherwise we'll have a sentry node being defined as a validator with a  comment to remind us about a special case which can be handled by its own var

Reply:
I'm actually skeptical whether a sentry will be able to produce blocks in shadow mode : 
https://github.com/maticnetwork/matic-cli/blob/f9706c3a6cbbf674094d61a854dd659833c18eb7/src/setup/devnet/index.js#L756
 
Need to double check.

Reply:
aaaah now I get what you mean, got it. Not about validator/sentry but ability or not to produce blocks.

Reply:
Indeed, boss!